firewall: add dmz and guests zone (#54)
NethServer/nethsecurity-docs: firewall.rst

Default zones can't be deleted but the network administrator can change existing policies or create new zones. 

Guests and DMZ zones
--------------------

In addition to the default zones, the firewall can be configured with additional zones to accommodate specific network requirements.

> Two common examples are the Guests and DMZ (Demilitarized Zone) zones.

Reviewer: ```suggestion
Two common examples are the Guest and DMZ (Demilitarized Zone) zones.
```

> Sometimes Guests zone is also named as vlue zone while DMZ zone is also named as orange zone.

Reviewer: ```suggestion
Sometimes the Guest zone is also known as the blue zone while the DMZ is also named as orange.
```

Guests zone (blue)
^^^^^^^^^^^^^^^^^^^

> Guests zone represents a network segment for guest devices, such as visitors or temporary users.

Reviewer: ```suggestion
The Guest zone represents a network segment for guest devices, such as visitors or temporary users.
```

This zone is typically isolated from the LAN zone to prevent unauthorized access to internal resources.
But it is allowed to access the WAN zone.

> To create a Guests zone, follow these steps:

Reviewer: ```suggestion
To create a Guest zone, follow these steps:
```

- access the ``Zones & policies`` section
- click on the ``Add zone`` button

> - enter **guests** inside the ``Name`` field: plesase note that the name is case sensitive and must be in lowercase, in this case the zone will be highlighted in blue

Reviewer: There's been some talks about naming the zone "guest" instead of guests.

Author: Where?

- leave empty the ``Allow forwards to`` field
- select ``LAN`` inside the ``Allow forwards from`` field
- enable the ``Traffic to WAN`` option
- select ``Drop`` for both ``Traffic to firewall`` and ``Traffic for the same zone`` fields
- click on the ``Save`` button and apply the changes

DMZ zone (orange)
^^^^^^^^^^^^^^^^^

> DMZ zone represents a network segment for servers and services that need to be accessible from the internet but isolated from the internal network.

Reviewer: ```suggestion
The DMZ represents a network segment for servers and services that need to be accessible from the internet but isolated from the internal network.
```

> To create a DMZ zone, follow these steps:

Reviewer: ```suggestion
To create a DMZ, follow these steps:
```
